parser/src/error.rs

```diff
@@ -24,7 +24,7 @@ pub enum ParseError {
     /// Parser encountered an unexpected token
     UnrecognizedToken(TokSpan, Vec<String>),
     /// Maps to `User` type from `lalrpop-util`
-    Other,
+    Other(LexicalError),
```

Comment:
It might be good to rename this error variant into `Lexical(LexicalError)`.

Reply:
Done